<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>Download</class>
  <widget class="QWidget" name="Download">
    <property name="windowModality">
      <enum>Qt::WindowModality::NonModal</enum>
    </property>
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>441</width>
        <height>73</height>
      </rect>
    </property>
    <property name="minimumSize">
      <size>
        <width>441</width>
        <height>73</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>441</width>
        <height>73</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>Fetching dependency...</string>
    </property>
    <property name="windowIcon">
      <iconset theme="applications-internet" />
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
        <widget class="QLabel" name="lb_progress">
          <property name="text">
            <string>?: 0/0 [00:00&lt;?, ?KB/s]</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QProgressBar" name="pb">
          <property name="value">
            <number>0</number>
          </property>
        </widget>
      </item>
    </layout>
  </widget>
  <resources />
  <connections />
</ui>